docs/integrations/sources/facebook-marketing.inapp.md

```diff
@@ -4,23 +4,36 @@
 
 ## Setup guide
 
+### For Airbyte Cloud:
+
+**To set up Facebook Marketing as a source in Airbyte Cloud:**
+
+1. [Log into your Airbyte Cloud](https://cloud.airbyte.io/workspaces) account.
+2. In the left navigation bar, click **Sources**. In the top-right corner, click **+ New source**.
+3. On the Set up the source page, select **Facebook Marketing** from the **Source type** dropdown.
+4. For Name, enter a name for the Facebook Marketing connector.
+
+**Facebook Marketing Source Settings:**
+
 1. Click **Authenticate your account** to authorize your [Meta for Developers](https://developers.facebook.com/) account. Airbyte will authenticate the account you are already logged in to. Make sure you are logged into the right account.
+2. Account ID: [Facebook Ad Account ID Number](https://www.facebook.com/business/help/1492627900875762): Open your Meta Ads Manager. Ad account ID number in the account dropdown menu or in your browser's address bar.
 2. For **Start Date**, enter the date in the `YYYY-MM-DDTHH:mm:ssZ` format. The data added on and after this date will be replicated. If this field is blank, Airbyte will replicate all data.
 
     :::warning
     Insight tables are only able to pull data from 37 months. If you are syncing insight tables and your start date is older than 37 months, your sync will fail.
     :::
 
 3. For **End Date**, enter the date in the `YYYY-MM-DDTHH:mm:ssZ` format. The data added on and before this date will be replicated. If this field is blank, Airbyte will replicate the latest data.
-4. For Account ID, enter your [Facebook Ad Account ID Number](https://www.facebook.com/business/help/1492627900875762).
-5. (Optional) Toggle the **Include Deleted** button to include data from deleted Campaigns, Ads, and AdSets.
+4. For **Access Token**, if you don't use OAuth. [Generate Access Token:](https://docs.airbyte.com/integrations/sources/facebook-marketing) The value of the access token generated. From your App’s Dashboard, click Marketing API --> Tools. Use permissions **ads_management, ads_read, read_insights, business_management**. And click Get token.
```

Comment:
Please update if you like the suggestion I made in the `spec.json` for this field.